Monika After Story/game/script-windowreacts.rpy

```diff
@@ -562,3 +562,1897 @@ label mas_wrs_twitch:
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_netscape')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_unity",
+            category=['unity'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_unity:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you designing your own game? Very cool!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_unity')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_gms2",
+            category=['gamemakerstudio2'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_gms2:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Are you designing your own game? Very cool!"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_gms2')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_sfm",
+            category=['sourcefilmmaker'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_sfm:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Making a video, [player]?"
+        ],
+        'Window Reactions'
+    )
+
+
+    if not wrs_success:
+        $ mas_unlockFailWRS('mas_wrs_sfm')
+    return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_windowreacts_database,
+            eventlabel="mas_wrs_norad",
+            category=['noradsantatracker'],
+            rules={
+                "notif-group": "Window Reactions",
+                "skip alert": None,
+                "keep_idle_exp": None,
+                },
+            show_in_idle=True
+        ),
+        code="WRS"
+    )
+
+label mas_wrs_norad:
+    $ wrs_success = mas_display_notif(
+        m_name,
+        [
+           "Hmmm, is it that time of year again? Happy Holidays, [player]!",
+           "I can't wait to spend the holiday with you tomorrow!"
```

Comment:
Wouldn't this one work no matter what date is it at the moment of reaction trigger?